perception_dataset/rosbag2/rosbag2_to_non_annotated_t4_converter.py

```diff
@@ -120,8 +124,8 @@ def __init__(self, params: Rosbag2ConverterParams) -> None:
         self._calibrated_sensor_target_frame: str = "base_link"
 
         # Note: To determine if there is any message dropout, including a delay tolerance of 10Hz.
-
-        self._TIMESTAMP_DIFF = params.timestamp_diff
+        # Note: The delay tolerance is set to 1.5 times the system scan period.
+        self._TIMESTAMP_DIFF = params.system_scan_period_sec * 1.5
```

Comment:
In alignment with this modification, could you change this all capital letter _TIMESTAMP_DIFF to small letter?

Reply:
Fixed: https://github.com/tier4/tier4_perception_dataset/pull/58/commits/bb99a0ec78b57df2dcfbb28436faef4831cbb758